doc/windows-package-manager-v1-roadmap.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-This document outlines our aspirational roadmap to delivering Windows Package Manager v1.0 by March-May 2021. We anticipate substantial feedback from the community, and as such, this plan is subject to change.
+This document outlines our aspirational roadmap to delivering Windows Package Manager v1.0 by Spring (March-May) 2021. We anticipate substantial feedback from the community, and as such, this plan is subject to change.
```

Comment:
Wasn't this explicitly removed in #354?

Reply:
Great catch! There was one "Spring" at the top so I merged them together. I'll remove Spring again.